modules/tabs-v2/assets/scss/frontend.scss

```diff
@@ -134,8 +137,7 @@
 	.elementor-tab-title,
 	.elementor-tab-title:before,
 	.elementor-tab-title:after,
-	.e-container,
-	.elementor-tabs-content-wrapper {
```

Comment:
Why delete this selector?

Reply:
That's a decision for product to make. Did you approve it with Aviv?

Reply:
@avivu 
By default the border width are 2pxs, I have changed that to 1px. Is that correct? That how it was in the existing Tabs widget.